### PR TITLE
Run the travis deploy on master only

### PR DESCRIPTION
**Pull request by Glowbal**
- Enable sqf validation on all branches
- Have the deploy script only run on the master branch.

### CODE REVIEW

**Comment by jonpas** [comment]
Is there a way to limit it to this repo only as well? All forks try to deploy it but of course Travis returns domain already taken.

**Comment by bux578** [comment]
there's the whitelist option http://docs.travis-ci.com/user/customizing-the-build/#Building-Specific-Branches

File: .travis.yml [hunk 1]
```diff
@@ -3,13 +3,25 @@ branches:
     - master
 language: python
 python:
-  - "3.4"
+- '3.4'
 before_script:
-  - pip install pygithub
-  - pip install pygithub3
+- if [ ${TRAVIS_BRANCH} = "master" ]; then [ -z "${GH_TOKEN}" ] && exit 0; [ ${TRAVIS_PULL_REQUEST}
```
Comment (jonpas) [inline .travis.yml]: Quotation marks around `${GH_TOKEN}` can be removed as well (works in my project using similar config).

File: .travis.yml [hunk 2]
```diff
@@ -3,13 +3,25 @@ branches:
     - master
```
Comment (jonpas) [inline .travis.yml]: Won't this make `sqf_validator.py` only ran on master as well? Or is that branch whitelist/blacklist different from PRs?

**Pull request merged**